iGCS/UserInterface/ViewControllers/WaypointMapBaseController.m

```diff
@@ -176,7 +180,9 @@ - (void) replaceMission:(WaypointsHolder*)mission {
     // Set the map extents
     NSUInteger offset = isHomeWPInitialised ? 0 : 1;
     if (offset < numWaypoints) {
```

Comment:
Merge two if statements, or swap ordering and put the declaration of offset in the outer if.